spec/src/main/asciidoc/overview.asciidoc

```diff
@@ -79,12 +79,52 @@ Instances of `ManagedExecutor` and `ThreadContext` can be injected into CDI bean
     unmanagedCompletionStage.thenApply(threadContext.contextualFunction(function3));
 ----
 
-The container provides default instances of `ManagedExecutor` and `ThreadContext`, as injected above, which are shared within an application with all other injection points for `ManagedExecutor` and `ThreadContext` that do not specify a qualifier. Their configuration is equivalent to `ManagedExecutor.builder().build()` and `ThreadContext.builder().build()`.
+The container allows to inject default instances of `ManagedExecutor` and `ThreadContext` which do not leverave either `@NamedInstance` or `ManagedExecutorConfig`/`ThreadContextConfig` annotations, as shown above.
+These instances are not shared and container creates a new instance for each injection point of `ManagedExecutor` or `ThreadContext`.
```

Comment:
Usage of the term 'shared' here is slightly confusing.  It is shared with the same injection point (for example, if in a request scoped bean that is created/destroyed often), but not with other injection points.  That said, given the requested update to the previous sentence, the current sentence should now be redundant and can be omitted.

Reply:
Agreed, that's a good reword you proposed